src/routes/manage_job/post.rs

```diff
@@ -0,0 +1,264 @@
+//! This file implements the `POST /bulk` endpoint.
+use crate::{
+	errors::ReacherError,
+	routes::check_email::{header::check_header, post::RetryOption},
+};
+use async_recursion::async_recursion;
+use check_if_email_exists::{
+	check_email as ciee_check_email, CheckEmailInput, CheckEmailInputProxy, CheckEmailOutput,
+	Reachable,
+};
+use sqlx::{Pool, Postgres};
+use warp::Filter;
+
+use std::{collections::HashMap, error::Error};
+
+use crate::routes::check_email::known_errors;
+use serde::{Deserialize, Serialize};
+use sqlxmq::{job, CurrentJob};
+
+const EMAIL_TASK_BATCH_SIZE: usize = 1;
```

Comment:
This is also confusing to me. Would it be possible to remove it, along with https://github.com/reacherhq/backend/pull/254#discussion_r791722062?

Reply:
I've added comments to explain how this is being used as the task size i.e. 1 task = EMAIL_TASK_BATCH_SIZE number of emails.

I've also added the two parts where changes will need to be made if this is to be increased to more than 1.